.github/workflows/test_project.yml

```diff
@@ -0,0 +1,76 @@
+name: Add issues and PRs to project and move issues on label change
+
+on:
+  issues:
+    types:
+      - opened
+      - labeled
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-issue-to-project:
+    name: Add issue to project with To Do status
+    if: github.event_name == 'issues' && github.event.action == 'opened'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
+        with:
+          project-url: https://github.com/orgs/pykale/projects/4
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          column-name: To do
+
+  add-pull-request-to-project:
+    name: Add pull request to project with In Progress status
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
+        with:
+          project-url: https://github.com/orgs/pykale/projects/4
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          column-name: In progress
+
+  move-issue-to-in-progress:
+    name: Move issue to In Progress column when work-in-progress label is added
+    if: github.event_name == 'issues' && github.event.action == 'labeled' && contains(github.event.issue.labels.*.name, 'work-in-progress')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move issue to In Progress column
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Duplicate necessary? See `env:` in https://github.com/pykale/pykale/blob/main/.github/workflows/project.yml